content/php-client/resources/product-entities/product-models.md

```diff
@@ -28,6 +28,10 @@ $client = new \Akeneo\Pim\ApiClient\AkeneoPimClientBuilder('http://akeneo.com/')
 $productModel = $client->getProductModelApi()->get('rain_boots_red');
 ```
 
+You can get more information about the returned format of the product values [here](/documentation/resources.html#product-values).
+
+In the Enterprise Edition, the response contains one more field `metadata`. Look at the [product model drafts](/php-client/ee-resources.html#product-model-draft) for an example.
```

Comment:
Precise that it's since the 2.3 version of the PIM

Reply:
fix